doc/aria-supported.md

```diff
@@ -8,6 +8,5 @@ For a detailed description about how accessibility support is decided, see [How
 
 ## Attributes
 
-| aria-attribute   | axe-core support |
-| ---------------- | ---------------- |
-| aria-describedat | No               |
+| aria-attribute | axe-core support |
+| -------------- | ---------------- |
```

Comment:
This seems odd. Should we turn off this part of the task? An empty table doesn't seem useful to me.

Reply:
I couldn't find any other docs page that had an empty table, so this does stand out. Since it is generated dynamically, would disabling the rule have unexpected consequences in the future? 

My thoughts now are to not disable toe task, but instead alter `aria-supported.js` to output an empty string if this might bite us later. If not, disabling the task or culling the code would certainly be easiest.

Reply:
Something needs to be done there. It's definitely outdated. Perhaps we should change that page to ARIA 1.3. I'll open a ticket.

Reply:
Ok, I'll let you open a ticket and will mark this as resolved as-is.

Reply:
New issue for this is #3590.